lib/sift/parameter.rb

```diff
@@ -8,35 +8,5 @@ def initialize(param, type, internal_name = param)
       @type = type
       @internal_name = internal_name
     end
-
```

Comment:
Parameter loses responsibility for parsing values because we want to parse things other than just the "chief" parameter.